src/shared_definitions.hpp

```diff
@@ -48,16 +56,16 @@
 enum class DbIndexType : uint8_t
 {
     FM_INDEX,
-    BI_FM_INDEX
+    BI_FM_INDEX,
 };
 
 inline std::string
 _indexEnumToName(DbIndexType const t)
 {
     switch (t)
     {
-        case DbIndexType::FM_INDEX:      return "fm_index";
-        case DbIndexType::BI_FM_INDEX:   return "bi_fm_index";
+        case DbIndexType::FM_INDEX:        return "fm_index";
+        case DbIndexType::BI_FM_INDEX:     return "bi_fm_index";
```

Comment:
undo whitespace changes, also the comman in line 59

Reply:
Mhm...Maybe I havn't figured it out. There are 41 characters before the `return ...` should be 40 or 44?